lib/aerospike/cluster.rb

```diff
@@ -584,7 +586,9 @@ def remove_alias(aliass)
     end
 
     def create_node(nv)
-      ::Aerospike::Node.new(self, nv)
+      node = ::Aerospike::Node.new(self, nv)
+      node.create_min_connections
```

Comment:
You can pass the number to the method. That would remove the need to open the cluster class variables to the node. It would also make that method more flexible.